lib/core/src/lib/header/navbar/navbar-item.component.html

```diff
@@ -0,0 +1,10 @@
+<button *ngIf="routerLink; else noRouterLink" class="adf-navbar-item-btn"
```

Comment:
can we avoid using CPU-heavy translate pipe here, and just use label? so that developers translate the text when providing the label from the outside?

Reply:
removed the dependency, ty